docs/bugpattern/ProtocolBufferOrdinal.md

```diff
@@ -13,5 +13,5 @@ You should very likely use `getNumber()` in preference to `ordinal()` in all
 circumstances since it's a more stable value.
 
 Note: If you're changing code that was already using ordinal(), it's likely that
-getNumber() will return a different real value. Tread carefully to avoid
+getNumber() will return a different real value. Treat carefully to avoid
```

Comment:
Likewise here :)

Reply:
Reverted to the original version, sorry for the noise!